packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -165,48 +315,18 @@ import { Alert, AlertActionCloseButton, AlertActionLink } from '@patternfly/reac
 </Alert>
 ```
 
-### Expandable
+### Live region alert examples
```

Comment:
The inclusion of live region examples for the alert component may need discussion, though I personally would lean towards omitting them on the Alert page. The updated a11y guidelines from https://github.com/patternfly/patternfly-org/pull/3040 doesn't include the `isLiveRegion` prop in the React table, since we recommend using Alert Group for alerts that need a live region.

Since two of the live region examples are also showing the live region functionality via the Alert Group, they feel more like demos than anything (this can be a followup issue). @mcarrano what do you think?

Reply:
I think my only concern with leaving it off is that users will want to have one single dynamic alert, not think to go to the AlertGroup page and will just conditionally render the Alert component on their own without seeing how to do it accessibly. I think it could be fine to leave off the example, as long as we consider how we're going to make sure that these users know how to add an accessible dynamic alert.

Reply:
Even if they have one single dynamic alert - they need to have the AlertGroup live region waiting to house that alert, right? If we want to keep the single alert example, I might advocate for using the AlertGroup in it to match our recommendation.

Reply:
That's a fair point @jessiehuff. Depending how we intend for users to use the Alert Group (only when more than 1 alert is used vs when at least 1 alert is used), I'd be fine with keep an example on the Alert page and going with Nicole's suggestion above of using AlertGroup in these Alert examples (which only the static live region example would need to be updated)